### PR TITLE
Update BUILDING.md

### PR DESCRIPTION
**Pull request by Haafingar**
### Contributor checklist

- [x] I have tested my contribution on these devices:
 * Device A, Android X.Y.Z
 * Device B, Android Z.Y
 * Virtual device W, Android Y.Y.Z
- [x] My contribution is fully baked and ready to be merged as is
- [x] I ensure that all the open issues my contribution fixes are mentioned in the commit message of my first commit using the `Fixes #1234` [syntax](https://help.github.com/articles/closing-issues-via-commit-messages/)

----------

### CODE REVIEW

File: BUILDING.md [hunk 1]
```diff
@@ -8,30 +8,26 @@ Session uses [Gradle](http://gradle.org) to build the project and to maintain
...
+You will then need to clone and run `./gradlew build` on each of the following repositories IN ORDER.
 
-        sdk.dir=/Application/android-sdk-macosx
-
-5. Using Java 8 
-
-6. Execute Gradle:
+https://github.com/loki-project/loki-messenger-android-curve-25519
+https://github.com/loki-project/loki-messenger-android-protocol
+https://github.com/loki-project/loki-messenger-android-meta
```
Comment (Mikunj) [inline BUILDING.md:27]: Actually the `protocol` and `meta` repos have been merged into `service` so we don't need to clone them

File: BUILDING.md [hunk 2]
```diff
@@ -45,20 +41,36 @@ pngs_from_svg.py ic_audio.svg /path/to/Session/res/ 150 --color #000 --opacity 0
...
-6. Paste the URL for the session-android project when prompted (https://github.com/loki-project/session-android.git).
-7. Android Studio should detect the presence of a project file and ask you whether to open it. Click "yes".
-9. Default config options should be good enough.
-9. Project initialisation and build should proceed.
+1. Open Android Studio. On a new installation, the Quickstart panel will appear. If you have open projects, close them using "File > Close Project" to see the Quickstart panel.
+2. From the Quickstart panel, choose "Checkout from Version Control" then "git".
+3. Paste the URL for the session-android project when prompted (https://github.com/loki-project/session-android.git).
+4. Android Studio should detect the presence of a project file and ask you whether to open it. Click "yes".
+5. Default config options should be good enough.
+6. Right now, we have kept the google-services.json file out of the public repository as it contains credentials for the firebase auth. We haven't yet made a way to build the app without this file or without firebase, as you don't technically need it to run the app. Email niels@loki.network to get this file for your build.
```
Comment (Mikunj) [inline BUILDING.md:54]: For this you can also make a dummy firebase application and get the json file from there. Should we list that as an option?

Comment (Haafingar) [reply BUILDING.md:54]: Yeah but I don't know how we'd do that, @nielsandriesse is seeing if it even matters if we just include the original file to the public or if there's anything sensitive

File: BUILDING.md (hunk 1 above)
Comment (Haafingar) [reply BUILDING.md]: https://github.com/loki-project/session-android/pull/201/commits/e653a21e37fc2276b17083961c53292f925978f8

**Comment by nielsandriesse** [comment]
This has been merged as part of #214

**Pull request closed without merging**